src/connectors/github_organization.py

```diff
@@ -0,0 +1,339 @@
+from json import dumps
+from time import sleep
+
+from runners.helpers import db
+from runners.helpers.dbconfig import WAREHOUSE
+
+S3_BUCKET_DEFAULT_PREFIX = ""
+
+CONNECTION_OPTIONS = [
+    {
+        'type': 'str',
+        'name': 'bucket_name',
+        'title': 'Github Organization Bucket',
+        'prompt': 'The S3 bucket Github Organization puts your logs',
+        'prefix': 's3://',
+        'placeholder': 'my-test-s3-bucket',
+        'required': True,
+    },
+    {
+        'type': 'str',
+        'name': 'filter',
+        'title': 'Prefix Filter',
+        'prompt': 'The folder in S3 bucket where Github Organization puts logs',
+        'default': S3_BUCKET_DEFAULT_PREFIX,
+        'required': True,
+    },
+    {
+        'type': 'str',
+        'name': 'aws_role',
+        'title': 'Github Organization Bucket Reader Role',
+        'prompt': "Role to be assumed for access to Github Organization files in S3",
+        'placeholder': 'arn:aws:iam::012345678987:role/my-github-reader-role',
+        'required': True,
+    }
+]
+
+FILE_FORMAT = """
+    TYPE = "JSON",
+    COMPRESSION = "AUTO",
+    ENABLE_OCTAL = FALSE,
+    ALLOW_DUPLICATE = FALSE,
+    STRIP_OUTER_ARRAY = TRUE,
+    STRIP_NULL_VALUES = FALSE,
+    IGNORE_UTF8_ERRORS = FALSE,
+    SKIP_BYTE_ORDER_MARK = TRUE
+"""
+
+LANDING_TABLE_COLUMNS = [
```

Comment:
Am I understanding right that this is describing a commit?

Reply:
What is describing a commit? The skip_byte_order_mark?

Reply:
I think Andrey is referring to the landing table itself; is each row of the table (and by extension, each event in the logs) a commit in github?

Reply:
https://developer.github.com/v3/activity/events/types/ there are a bunch of webhook types other than commits

Reply:
Got it. The comment feature on GitHub shows 3 lines before the one I am commenting on, so I was referring to the `LANDING_TABLE_COLUMNS`.

Do I understand right that we're landing all of the webhooks into a table that has the union of all of the properties in each of them? That seems not ideal, but OK, as long as we make this clear to someone looking at the connector (if possible), or when reading the readme.